add german translation

arbor95 opened the pull request:
see web.bild-raum.de for working translation.

arbor95 committed:
add german translation

vytux-com reviewed (commented):
I think this is almost there but the last string is not right.

I think German language is more like English or Danish when it comes to plural translations and should have 1 singular string and 1 plural string.

Currently you have one translation on 349 and 2 Lithuanian translations (350, 351) those 2 should be deleted and replace with 1 German one

arbor95 committed:
upload the correct module.php.
change singular translation for "has one ..."
remove lines from copy/pasted/renamed lithuanianTranslations

arbor95 committed:
Vetter, Kusine -> Cousins, Cousinen

vytux-com commented:
For some reason, I am now seeing the whole file being changed... I am trying to figure out what has changed on every line

arbor95 commented:
A change in every line will surely be the change of Linefeed to CR LF (\r\n) cause I work with Windows, sorry.
